### PR TITLE
Create gain factor keyword schema.

### PR DESCRIPTION
**Pull request by dmggh**
Gain factor keyword is needed for the read noise schema ( RCAL-77 and PR #114 ). This is supported by PR #118.

### CODE REVIEW

**Review by ddavis-stsci — APPROVED** [review]
Besides needing a "return" at the end it looks good.

File: romancal/datamodels/schemas/reference_files/keyword_gainfact.schema [hunk 1]
```diff
@@ -0,0 +1,12 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/asdf/asdf-schema-1.0.0"
+id: "http://stsci.edu/schemas/roman_datamodel/reference_files/keyword_gainfact.schema"
+type: object
+properties:
+  meta:
+    type: object
+    properties:
+      gain_factor:
```
Comment (eslavich) [inline romancal/datamodels/schemas/reference_files/keyword_gainfact.schema:10]: Does it make sense for this value to be negative?  Could add:

```yaml
gain_factor:
  title: Gain scale factor
  type: number
  minimum: 0.0
```
or if zero is also not permitted:
```yaml
gain_factor:
  title: Gain scale factor
  type: number
  minimum: 0.0
  exclusiveMinimum: true
```

File: romancal/datamodels/schemas/reference_files/keyword_gainfact.schema [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+---
+$schema: "http://stsci.edu/schemas/asdf/asdf-schema-1.0.0"
+id: "http://stsci.edu/schemas/roman_datamodel/reference_files/keyword_gainfact.schema"
+type: object
+properties:
+  meta:
+    type: object
+    properties:
+      exposure:
+      type: object
```
Comment (nden) [inline romancal/datamodels/schemas/reference_files/keyword_gainfact.schema:11]: Indentation is incorrect here - this and the following  line need to be indented.

**Comment by dmggh** [comment]
Closing, as these reviewer comments were addressed in PR#134.

**Pull request closed without merging**